sql/database.sql

```diff
@@ -5031,12 +5031,7 @@ CREATE TABLE `patient_data` (
   `billing_note` text,
   `homeless` varchar(255) NOT NULL default '',
   `financial_review` datetime default NULL,
-  `pubpid` varchar(255) NOT NULL default '',
```

Comment:
Will break code base if removed

Reply:
...then I want to figure out where and why.  We agreed to get away from pubpid a long time ago, and I have not used it for years.

Reply:
Just a few
![image](https://user-images.githubusercontent.com/4655366/31405412-fb5cc100-adcc-11e7-995f-7e3c1f50c8a5.png)